tools/cli/commands.go

```diff
@@ -500,6 +510,11 @@ func startWorkflowHelper(c *cli.Context, shouldPrintProgress bool) {
 	if c.IsSet(FlagCronSchedule) {
 		startRequest.CronSchedule = common.StringPtr(c.String(FlagCronSchedule))
 	}
+	memoStr := processJSONMemo(c)
+	if len(memoStr) != 0 {
+		fields := map[string][]byte{memoKeyForCLI: []byte(processJSONMemo(c))}
```

Comment:
I think the key should be passed as a parameter.

Reply:
I think this is simplest for user to use. 
An alternative is: provide one optional "memo-key" in start (but now user have to put same key when list)